From 3c7d49d95be71c26afdbaef14def92f3460c7373 Mon Sep 17 00:00:00 2001
From: Dan Church <h3xx@users.noreply.github.com>
Date: Mon, 29 Dec 2025 05:09:04 -0600
Subject: [PATCH] Fix detection of git clone (#99)

If it's cloned as a git submodule, .git will be a single-line file, not
a directory.
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 6504bf7..d85a3ec 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -20,7 +20,7 @@ NEWS:
 	[ -f NEWS.md ] && cat $(top_srcdir)/NEWS.md > $@ || true
 
 ChangeLog:
-	[ -d .git ] && git log > ChangeLog || true
+	[ -e .git ] && git log > ChangeLog || true
 
 pkgdocdir = @docdir@
 pkgdoc_DATA = \
